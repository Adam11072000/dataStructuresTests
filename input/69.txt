init 211
mergeSuperPixels 138 213 132
addImage -4
addImage -8
resetLabelScore 139 120 -8
setLabelScore 129 129 156 15
getHighestScoredLabel 57 -7
addImage -6
addImage 149
quit