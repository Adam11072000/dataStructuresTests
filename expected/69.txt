init done.
mergeSuperPixels: INVALID_INPUT
addImage: INVALID_INPUT
addImage: INVALID_INPUT
resetLabelScore: INVALID_INPUT
setLabelScore: FAILURE
getHighestScoredLabel: INVALID_INPUT
addImage: INVALID_INPUT
addImage: SUCCESS
quit done.
